### DISCUSSION TITLE
Uploaded a Image on Social Preview but cannot see any

### DISCUSSION THREAD

**Discussion by BurhanSiraj** [community/community#121294] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I uploaded a Image showing my Small-Project preview in my Repositry settings tab in Social Preview section which means to show all users a preview image of my project that how does it looks like but cannot be seen anywhere in any Repositry. It can seen only by me in my settings tab of the Repositry but not by any other users. If that Social Preview is not that what I am thinking then for what it is but if it is what I am thinking then why my Image is not being showed. **Please Help**

![Screenshot_1](https://github.com/community/community/assets/129822847/0c9c73e5-dda2-4e62-8abe-a35a1a58ce9e)

**Comment by gurjeetsinghvirdee** [comment]
hey @BurhanDon 

You have the ability to personalize the image showcased on social media platforms when your repository link is shared by uploading an image in the repository settings. This image, commonly known as a social preview or social share image, acts as a visual representation of your project..

By uploading a customized image in your repository settings, whenever your project link is shared on social media, it will be accompanied by the image you've selected. This ensures that your project is presented in a visually appealing manner across various platforms, helping to attract more viewers and engagement.

**Like this:** 
![image](https://github.com/community/community/assets/73753957/4ab74bb1-e6b1-4c39-ab43-a11cd18cf5d8)

**Comment by davidhammad** [comment]
i do he same but still not working for my [top follow apk](https://topfollowerpro.com)
